### DISCUSSION TITLE
🐞 Wide Modal

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3243] [Help: Figma] [answered]
Hi! It appears as though the Modal component isn't letting me switch the width from default to wide. I can't tell if this is a bug, or if i'm doing something wrong. thanks!

**Comment by loreina** [comment]
Hi @ceyerkes,

Can you share a loom/video of the issue? And which Figma library you're using? I can't seem to replicate it. Thanks!

**Reply by ceyerkes** [reply]
I'm using the [NEW] Paste Components Library. 

https://github.com/twilio-labs/paste/assets/87141551/e62305ed-0368-4313-9ad6-110ea63eb27a

Thanks!

**Reply by loreina** [reply]
Thanks! We have a ticket in our backlog to investigate the bug and will follow up here on the resolution.

**Comment by serifluous** [comment, marked as answer]
Just merged in and published the fix for this!
